icons/folder_type_app.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>folder_type_app</title><path d="M27.5,5.5H18.2L16.1,9.7H4.4V26.5H29.6V5.5Zm0,4.2H19.3l1.1-2.1h7.1Z" style="fill:#14622a"/><path d="M10,12.983V31.017H31V12.983ZM30.219,30.236H10.781V15.247H30.219Z" style="fill:#7bdbab"/><path d="M19.981,17.138c.352-.022.768-.15,1.071.107.277.255.264.669.352,1.009a6.92,6.92,0,0,1,1.234.393c.26-.229.486-.564.856-.6.329-.023.567.235.8.425a3.072,3.072,0,0,1,.685.679c.168.445-.2.822-.422,1.158a7.688,7.688,0,0,1,.55,1.1c.431.027,1.038-.07,1.241.425a5.126,5.126,0,0,1,.142,1.492c-.11.531-.743.551-1.174.646-.072.393-.235.76-.342,1.143.117.336.533.5.612.872.085.405-.275.693-.5.981-.216.228-.39.56-.727.621-.438.079-.754-.295-1.109-.479a6.271,6.271,0,0,1-1.1.564c-.031.452.082,1.117-.452,1.316a5.036,5.036,0,0,1-1.518.117c-.525-.168-.494-.8-.612-1.242-.39-.072-.748-.256-1.134-.329-.352.249-.639.759-1.141.631a3.557,3.557,0,0,1-.851-.627c-.207-.186-.481-.38-.47-.69-.021-.412.392-.666.47-1.04-.14-.338-.352-.643-.486-.988-.366-.033-.784.065-1.1-.16-.295-.2-.278-.6-.321-.91a2.958,2.958,0,0,1,.012-.97c.207-.438.759-.416,1.161-.531a6.482,6.482,0,0,1,.352-1.174c-.235-.276-.574-.5-.655-.878-.046-.345.235-.6.432-.849a2.7,2.7,0,0,1,.7-.7c.44-.146.794.22,1.134.433a5.911,5.911,0,0,1,1.157-.587c.093-.375-.053-.845.269-1.128C19.33,17.129,19.682,17.178,19.981,17.138Zm.117,3.828a2.144,2.144,0,1,0,1.908.566,2.144,2.144,0,0,0-1.908-.566Z" style="fill:#7bdbab"/></svg>
```

Comment:
![image](https://user-images.githubusercontent.com/696981/31854884-46485cf2-b6a0-11e7-817c-4ee5cff2d172.png)

Could we provide some color background to the sub-icon to avoid this transparency effect?

Reply:
Never mind. I've tested it and I prefer it the way it is now. 😉